### ISSUE TITLE
Use fastest-levenshtein

### ISSUE THREAD

**Issue by jfmengels** [jfmengels/node-elm-review#82]
I see that our dependency https://www.npmjs.com/package/fast-levenshtein uses https://www.npmjs.com/package/fastest-levenshtein under the hood. Maybe that one would be enough to use use a dependency.

Maybe we could/should inline it as well?

**Comment by marc136** [comment]
I had a short look at the code of `fast-levenshtein`, and replacing it with `fastest-levenshtein` should not be a problem.

elm-review uses `levenshtein.get(flag, f.name)` and does not pass the option to use locale-sensitive string comparisons. This means that `fast-levenshtein` does this:

```js
var levenshtein = require('fastest-levenshtein');
...
var Levenshtein = {
    get: function(str1, str2, options) {
        if (options.collate) { ... }
        return levenshtein.distance(str1, str2);
    }
    ...
}
```
See https://github.com/hiddentao/fast-levenshtein/blob/master/levenshtein.js#L84

Inlining `fastest-levenshtein` should pose no problem, because elm-review only needs 3 of the 4 functions (about 130 lines) and the code is MIT-licensed.

**Comment by lishaduck** [comment]
Closed by #130.
